### ISSUE TITLE
What is SteamCAD2?

### ISSUE THREAD

**Issue by ghost** [oskardolch/SteamCAD2#1]
Do you working on enhanched version of `SteamCAD`?

- https://github.com/oskardolch/SteamCAD

**Comment by oskardolch** [comment]
Yes, I am working on a new version, which should allow to define paths,
areas and colors. But it is still less than an Alpha version.
